### PR TITLE
[Merged by Bors] - Integrate API v1.9.0

### PR DESCRIPTION
**Pull request by fasmat**
## Motivation
Integrate the changes from https://github.com/spacemeshos/api/pull/230

## Changes
- API now uses uint32 for epochs instead of uint64
- Use Layernum and Epochnum where applicable int he API

## Test Plan
n/a

## TODO

- [x] Explain motivation or link existing issue(s)
- [x] Test changes and document test plan
- [x] Update documentation as needed

## DevOps Notes

- [x] This PR does not require configuration changes (e.g., environment variables, GitHub secrets, VM resources)
- [x] This PR does not affect public APIs
- [x] This PR does not rely on a new version of external services (PoET, elasticsearch, etc.)
- [x] This PR does not make changes to log messages (which monitoring infrastructure may rely on)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -33,7 +33,7 @@ require (
 	github.com/pyroscope-io/pyroscope v0.37.2
 	github.com/santhosh-tekuri/jsonschema/v5 v5.3.0
 	github.com/seehuhn/mt19937 v1.0.0
-	github.com/spacemeshos/api/release/go v1.8.1-0.20230417051650-d4cb54e6c592
+	github.com/spacemeshos/api/release/go v1.8.1-0.20230415112814-1ab9622d64ad
```
Comment (fasmat) [inline go.mod:36]: TODO: updated after https://github.com/spacemeshos/api/pull/230 is merged

**Comment by fasmat** [comment]
bors try

**Comment by fasmat** [comment]
bors try

**Review by dshulyak — APPROVED** [review]

**Comment by fasmat** [comment]
bors merge

**Comment by fasmat** [comment]
bors merge

**Comment by countvonzero** [comment]
bors merge

**Pull request closed without merging**